services/notifications/pkg/service/service.go

```diff
@@ -53,6 +54,21 @@ func (s eventsNotifier) Run() error {
 							Str("event", "ShareCreated").
 							Msg("failed to send a message")
 					}
+				case events.VirusscanFinished:
+					if !e.Infected {
+						// no need to annoy the user
+						return
+					}
+
+					if e.ExecutingUser == nil {
+						s.logger.Error().Str("events", "VirusscanFinished").Str("uploadid", e.UploadID).Msg("no executing user")
+						return
+					}
+					m := "Dear %s,\nThe virusscan of file '%s' discovered it is infected with '%s'.\nThe system is configured to handle infected files like: %s.\nContact your administrator for more information."
```

Comment:
fine for now but later we need it in other languages too.... out of scope for now

Reply:
Yes absolutely. There is a ticket for that already somewhere. Propbably in Jira.